packages/components/src/highlight-cards/highlight-card.tsx

```diff
@@ -27,24 +30,37 @@ export function percentCalculator( part: number | null, whole: number | null ) {
 	return Math.abs( answer ) === Infinity ? 100 : Math.round( answer );
 }
 
+const FORMATTER = new Intl.NumberFormat();
+function formatNumber( number: number | null ) {
```

Comment:
We could modify and export the [existing formatNumber](https://github.com/Automattic/wp-calypso/blob/0ef4f96bd3b1c0cea1e3effb407fca54c3adba8e/packages/components/src/shortened-number/index.tsx#L12-L14) to reuse here, which also benefits the [usage inside AllTimeHighlightsSection](https://github.com/Automattic/wp-calypso/blob/0ef4f96bd3b1c0cea1e3effb407fca54c3adba8e/client/my-sites/stats/all-time-highlights-section/index.tsx#L35-L38) in the future.

Reply:
The first one you mentioned outputs a specific short version; if we extend it, its name won't be accurate anymore. 
We could create a utility file inside the stats directory and import the function from there. Do you think this would improve the usage here? If you think so, I could add it in another PR.

Reply:
> We could create a utility file inside the stats directory and import the function from there.

Sounds like a great plan. We could have more discussion on that PR later.